frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx

```diff
@@ -3,97 +3,139 @@ import * as _ from 'lodash';
-          } else {
-            setFieldValue('isSearchingForImage', false);
-            setFieldValue('isi', {});
-            setFieldValue('isi.status', status.message);
-            setFieldValue('route.targetPort', null);
-          }
-        })
-        .catch((error) => {
-          setFieldValue('isi', {});
-          setFieldValue('isi.status', error.message);
+    k8sCreate(ImageStreamImportsModel, importImage)
+      .then((imageStreamImport) => {
+        const status = _.get(imageStreamImport, 'status.images[0].status');
+        if (status.status === 'Success') {
+          const name = _.get(imageStreamImport, 'spec.images[0].from.name');
+          const image = _.get(imageStreamImport, 'status.images[0].image');
+          const tag = _.get(imageStreamImport, 'status.images[0].tag');
+          const isi = { name, image, tag, status };
+          const ports = getPorts(isi);
           setFieldValue('isSearchingForImage', false);
-        });
-    },
-    [setFieldValue, values.application.name, values.name, values.project.name],
-  );
+          setFieldValue('isi.name', name);
+          setFieldValue('isi.image', image);
+          setFieldValue('isi.tag', tag);
+          setFieldValue('isi.status', status);
+          setFieldValue('isi.ports', ports);
+          setFieldValue('image.ports', ports);
+          setFieldValue('image.tag', tag);
+          !values.name && setFieldValue('name', getSuggestedName(name));
+          !values.application.name &&
+            setFieldValue('application.name', `${getSuggestedName(name)}-app`);
+          // set default port value
+          const targetPort = _.head(ports);
+          targetPort && setFieldValue('route.targetPort', makePortName(targetPort));
+          setValidated(ValidatedOptions.success);
+        } else {
+          setFieldValue('isSearchingForImage', false);
+          setFieldValue('isi', {});
+          setFieldValue('isi.status', status.message);
+          setFieldValue('route.targetPort', null);
+          setValidated(ValidatedOptions.error);
+        }
+      })
+      .catch((error) => {
+        setFieldValue('isi', {});
+        setFieldValue('isi.status', error.message);
+        setFieldValue('isSearchingForImage', false);
+        setValidated(ValidatedOptions.error);
+      });
+  }, [setFieldValue, values.application.name, values.name, values.project.name, values.searchTerm]);
 
   const handleSave = (name: string) => {
     setNewImageSecret(name);
-    values.searchTerm && handleSearch(values.searchTerm);
+    values.searchTerm && handleSearch();
+  };
+
+  const getHelpText = () => {
+    if (values.isSearchingForImage) {
+      return (
+        <span>
+          <LoadingInline />
+        </span>
+      );
+    }
+    if (!values.isSearchingForImage && validated === ValidatedOptions.success) {
+      return (
+        <strong className="odc-image-search_help-text-success">
```

Comment:
This shouldn't be bold and the styling should be set correctly by patternfly. Only need the icon and text.

Reply:
This is bold to keep it consistent with git-flow. We use bold text there for validated text.

Reply:
removed stylesheet